utils/jwtUtil.js

```diff
@@ -141,7 +159,14 @@ function verifyUserToken(req, cb) {
       });
     }
 
-    req.user = user;
+    req.user = user.get();
+    return Profile.isAdmin(req.user.profileId);
+  })
+  .then((isAdmin) => {
+    assignKeyValue(req.headers, 'userName', req.user.name);
+    assignKeyValue(req.headers, 'tokenName', decodedData.tokenname);
+    assignKeyValue(req.headers, 'profileName', req.user.profile.name);
+    assignKeyValue(req.headers, 'isAdmin', isAdmin);
```

Comment:
needs "IsCollector" set to false - for all three situations (bot, user, collector) we need to set ALL of these new named headers so that we are certain to be overwriting something the user might send in to try to spoof us

Reply:
Yes. Makes sense.